cumulusci/tasks/apex/testrunner.py

```diff
@@ -330,6 +330,13 @@ def _get_test_results(self, allow_retries=True):
                 f"Class {class_name} failed to run some tests with the message {error}. Applying error to unit test results."
             )
 
+            # In Spring '20, we cannot get symbol tables for managed classes.
```

Comment:
@davidmreed didn't it appear to be an issue with getting the symbol table for any class with a namespace prefix, whether managed or not?

Reply:
Yes, but the response from Core was that that appears to be a bug rather than the intended behavior.

The test runner does not discriminate between namespaced and managed, so this should hit both. The only downside is that we'll in after Core fixes the bug treat namespaced runs like managed, but a larger refactor would be needed to fix that.

Reply:
Ah yes, I see how we are using the `managed` option for unmanaged namespaced feature builds as well.